### DISCUSSION TITLE
RDS using

### DISCUSSION THREAD

**Discussion by vsudenko** [TPC-Council/HammerDB#171] [Q&A] [answered]
Hello Gents,

I have a question about using HammerDB on RDS MSSQL instance. Are there special settings required? I cannot find the document about RDS testing.

Thanks,

Regards,
Vadim

**Comment by sm-shaw** [comment, marked as answer]
To be clear you mean Amazon RDS? (There are also other database services called RDS such as Alibaba, Huawei etc). There is no explicit documentation for AWS RDS and it is known to work from the many publications so there should not be particular issues with MSSQL on AWS RDS. Having said that the documentation is open source in Docbook format located here https://github.com/TPC-Council/HammerDB/blob/master/DocBook/docs.xml and therefore anyone is able to update this and submit a pull request if there is specific RDS related information that would benefit and they wish to contribute.

**Comment by vsudenko** [comment]
Yes, I am talking about Amazon RDS. Thanks for info.
